src/Tribe/Settings_Manager.php

```diff
@@ -163,6 +163,20 @@ public static function set_option( $name, $value ) {
 		return self::set_options( $options );
 	}
 
+	/**
+	 * Remove an option
+	 *
+	 * @param string $name
+	 *
+	 * @return bool
+	 */
+	public static function remove_option( $name ) {
+		$options          = self::get_options();
+		unset( $options[ $name ] );
+
+		return self::set_options( $options );
```

Comment:
Use static?

Reply:
I used self because of set option above, but yes.